### ISSUE TITLE
when no avatar image exists, errors are logged to the console

### ISSUE THREAD

**Issue by McPizza0** [uninbox/UnInbox#33]
when a user, org, group or contact does not have an avatar image, an error is logged out to the console from the Avatar component
![image](https://github.com/uninbox/UnInbox/assets/17185737/141d654f-4c08-4a72-b4e4-5ce5f32debb4)
https://github.com/nuxt/ui/blob/dev/src/runtime/components/elements/Avatar.vue

we do not know if an object has an avatar image or not
so we always pass a URL to the avatar component, which tries to fetch the avatar

we need a way to suppress these error logs, or hide them in another way

if we cant remove them, we may need to add a `hasAvatar` boolean to the profile tables for each object type

@cstrnt thoughts

**Comment by cstrnt** [comment]
I would honestly just add a field to each model in the db schema like `avatarUrl` which can be nullable.
This allows us to not fetch the avatar if there isn't one or just default to some other URL :)

**Comment by skushagra9** [comment]
@cstrnt are you working on this ??

**Comment by cstrnt** [comment]
@skushagra9 nope, I think we should still discuss a little about the implementation and its implications before we do that. Feel free to leave your thoughts here

**Comment by McPizza0** [comment]
> I would honestly just add a field to each model in the db schema like avatarUrl which can be nullable.
This is how we had it when using cloudflare images
switched to current method to ensure that unused (replaced) avatars dont take up unnecessary storage space

we could achieve the same as your proposal by having `avatar = true` in the db instead of a url
and pass that to the avatar component

**Comment by cstrnt** [comment]
That's right, both solutions have advantages and disadvantages. I personally lean towards the solution of storing the url in the db (reference: [dub](https://github.com/dubinc/dub/blob/c74c8b54bd607fff78243d3843e6035eab2bf830/apps/web/prisma/schema.prisma#L20)

**Comment by McPizza0** [comment]
updated issue description

**Comment by cstrnt** [comment]
> between 

Agree! We could prefix the nanoId with the type so it ends up like <type>_<nanoId>. Unkey is doing that (as well as stripe) and I love that

**Comment by McPizza0** [comment]
> prefix the nanoId with the type so it ends up like _. 

good idea, updated issue description
we good to go @cstrnt ?

**Comment by cstrnt** [comment]
Looks good to me. Should be scoped well enough for implementation

**Comment by skushagra9** [comment]
Can I work on this ?
